### PR TITLE
Upgrade openjdk (7 to 8) version in Docker build

### PR DESCRIPTION
**Pull request by 0asa**

### CODE REVIEW

File: app/controllers/Application.scala [hunk 1]
```diff
@@ -659,6 +659,7 @@ object Application extends Controller {
             }
             Ok(json).withHeaders(
               HeaderNames.CONTENT_DISPOSITION → s"""attachment; filename="$path" """,
+              HeaderNames.CONTENT_TYPE → "application/force-download",
```
Comment (vidma) [inline app/controllers/Application.scala]: this commit shouldn't be in this PR, right?

Comment (0asa) [inline app/controllers/Application.scala]: maybe

**Comment by andypetrella** [comment]
@frbayart do you think this one can be 🚢 ?

**Comment by frbayart** [comment]
GO

**Comment by 0asa** [comment]
This should close issue #734 I think...

**Pull request merged**